.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+  "settings": {
+    "jest": {
+      "version": ">= 27"
```

Comment:
🔶 This is only necessary when using something other than `jest` and we need to tell the Jest lint rules what version of Jest's syntax we are using.

(e.g. vitest in the template project)